pkg/crc/preflight/preflight_darwin.go

```diff
@@ -23,6 +23,12 @@ func StartPreflightChecks() {
 		"Checking file permissions for resolver",
 		config.GetBool(cmdConfig.WarnCheckResolverFilePermissions.Name),
 	)
+
+	preflightCheckSucceedsOrFails(config.GetBool(cmdConfig.SkipCheckResolvConfFilePermissions.Name),
+		checkResolvConfFilePermissions,
+		"Checking file permissions for resolv.conf",
```

Comment:
We should put the full path of the file `/etc/resolv.conf` and `/etc/resolver/testing`